lib/averages.js

```diff
@@ -0,0 +1,52 @@
+var pry = require('pryjs');
+
+Array.prototype.total = function() {
+   var finalTotal = this.reduce(function(total, number){
+     return total + number;
+   }, 0);
+   return finalTotal
+}
+
+Array.prototype.mean = function() {
+  return (this.total()) / (this.length);
+}
+
+Array.prototype.median = function() {
+  if((this.length - 1) % 2 == 0){
+    return this[(this.length - 1) / 2];
+  }
+  else {
+    var eOne = this[Math.floor((this.length - 1) / 2)]
+    var eTwo = this[Math.ceil((this.length - 1) / 2)]
+    return (eOne + eTwo) / 2
+  }
+}
+
+Array.prototype.mode = function() {
```

Comment:
this function is pretty long, could we make this more single responsibility and break it down?